<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileExistsDialog</class>
 <widget class="QWidget" name="fileExistsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The file exists</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Destination</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_source_size">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_content_source_size">
         <property name="text">
          <string notr="true">0 KiB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_source_modified">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modified</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_content_source_modified">
         <property name="text">
          <string notr="true">Today</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_source_file_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_content_source_file_name">
         <property name="text">
          <string notr="true">source.txt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_content_source_folder">
         <property name="text">
          <string notr="true">/source/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_destination_size">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_content_destination_size">
         <property name="text">
          <string notr="true">0 KiB</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_destination_modified">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modified</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_content_destination_modified">
         <property name="text">
          <string notr="true">Today</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_destination_file_name">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_content_destination_file_name">
         <property name="text">
          <string notr="true">destination.txt</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_content_destination_folder">
         <property name="text">
          <string notr="true">/destination/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEditNewName">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SuggestNewName">
       <property name="text">
        <string>Suggest new &amp;name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxAlways">
       <property name="text">
        <string>&amp;Always perform this action</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Rename">
       <property name="text">
        <string>&amp;Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="Overwrite">
       <property name="text">
        <string>&amp;Overwrite</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Skip">
       <property name="text">
        <string>&amp;Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionOverwrite_if_newer">
   <property name="text">
    <string>Overwrite if newer</string>
   </property>
  </action>
  <action name="actionOverwrite_if_not_same_modification_date">
   <property name="text">
    <string>Overwrite if modification date differs</string>
   </property>
   <property name="toolTip">
    <string>Overwrite if modification date differs</string>
   </property>
  </action>
  <action name="actionOverwrite_if_older">
   <property name="text">
    <string>Overwrite if older</string>
   </property>
   <property name="toolTip">
    <string>Overwrite if older</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>toggled(bool)</signal>
   <receiver>Cancel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditNewName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAlways</sender>
   <signal>toggled(bool)</signal>
   <receiver>SuggestNewName</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
